apps/site/assets/ts/schedule/components/schedule-finder/upcoming-departures/TripStop.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { ReactElement } from "react";
+import { alertIcon } from "../../../../helpers/icon";
+import { breakTextAtSlash } from "../../../../helpers/text";
+import { isSkippedOrCancelled } from "../../../../models/prediction";
+import { TripDepartureWithPrediction } from "../../__trips";
+
+interface Props {
+  departure: TripDepartureWithPrediction;
+  index: number;
+  showFare: boolean;
+  routeType: number;
+}
+
+const formattedDepartureTimes = (
+  departure: TripDepartureWithPrediction,
+  routeType: number
+): ReactElement<HTMLElement> => {
+  const { schedule, prediction, delay } = departure;
+  if (routeType === 2) {
```

Comment:
We have `isACommuterRailRoute` for better readability.

Reply:
Good point. I missed that at first because that function took a Route not a RouteType which is all we have here, but I updated it to take either.